specification/azurefleet/AzureFleet.Management/fleet.tsp

```diff
@@ -211,6 +228,213 @@ union EvictionPolicy {
+
+  @doc("The range of network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  networkInterfaceCount: VMAttributeMinMax;
+
+  @doc("The range of network bandwidth in Mbps specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  networkBandwidthInMbps: VMAttributeMinMax;
+
+  @doc("Specifies whether the VMSize supporting RDMA (Remote Direct Memory Access) should be used to build Fleet or not.")
+  rdmaSupport?: VMAttributeSupport;
+
+  @doc("The range of RDMA (Remote Direct Memory Access) network interface count specified from Min to Max. Optional parameter. Either Min or Max is required if specified.")
+  rdmaNetworkInterfaceCount: VMAttributeMinMax;
+
+  @doc("Specifies whether the VMSize supporting GPU should be used to build Fleet or not.")
+  gpuSupport?: VMAttributeSupport;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The GPU manufacturers specified as a list. gpuSupport should be set to Included or Required to use this VMAttribute. Default GPU manufacturers if not specified are ANY of the valid values.")
+  gpuManufacturers: Array<GpuManufacturer>;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The VM category specified as a list. Optional parameter.")
+  vmCategories: Array<VMCategory>;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The VM architecture types specified as a list. Optional parameter.")
+  architectureTypes: Array<ArchitectureType>;
+
+  @extension("x-ms-identifiers", [])
+  @doc("The VM CPU manufacturers specified as a list. Optional parameter.")
+  cpuManufacturers: Array<CpuManufacturer>;
+
+  @doc("Specifies whether the VMSize supporting burstable capability should be used to build Fleet or not.")
+  burstableSupport?: VMAttributeSupport;
+
+  @extension("x-ms-identifiers", [])
+  @doc("Specifies which VMSizes should be excluded while building Fleet. Optional parameter.")
+  excludedVMSizes: Array<string>;
+}
+
+@added(Versions.v2024_11_01)
+@doc("Represents the profile for a single additional location in the Fleet. The location and the (optional).")
+model LocationProfile {
+  @doc("The ARM location name of the additional region.")
+  location: string;
+
+  @doc("""
+    Base Virtual Machine Profile Properties to be specified according to "specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/{computeApiVersion}/virtualMachineScaleSet.json#/definitions/VirtualMachineScaleSetVMProfile"
+    """)
+  virtualMachineProfileOverride: BaseVirtualMachineProfile;
+}
+
+@added(Versions.v2024_11_01)
+@doc("Represents the configuration for additional locations where Fleet resources may be deployed.")
+model AdditionalLocationsProfile {
+  @extension("x-ms-identifiers", [])
+  @doc("The list of location profiles.")
+  locationProfiles: Array<LocationProfile>;
+}
+
+#suppress "@azure-tools/typespec-azure-core/casing-style" "VMAttributesList which does not follow camel casing"
```

Comment:
Where is `VMAttributesList` is being used?

Reply:
removed.